frontend/components/card/search-result/CardSearchResultUser.vue

```diff
@@ -23,21 +17,15 @@
             <Icon name="ph:dots-three-circle-vertical" size="1.75em" />
           </div>
         </div>
-        <div
-          class="items-center hidden w-full md:flex md:w-fit space-x-3 lg:space-x-5"
-        >
+        <div class="items-center hidden w-full md:flex md:w-fit space-x-3 lg:space-x-5">
           <MetaTagLocation :location="user?.location" />
         </div>
       </div>
       <div class="flex flex-col md:flex-row space-y-3 md:space-y-0">
-        <div
-          class="flex items-center justify-center w-full md:hidden space-x-3"
-        >
+        <div class="flex items-center justify-center w-full md:hidden space-x-3">
           <MetaTagLocation :location="user?.location" />
         </div>
-        <div
-          class="flex justify-center space-x-3 lg:space-x-4 md:justify-start"
-        >
+        <div class="flex justify-center space-x-3 lg:space-x-4 md:justify-start">
```

Comment:
Similar note on the formatting of the class here :) Seems to be problematic given it's not showing up as a string :)